recipes/anuga/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "anuga" %}
+{% set version = "3.1.8" %}
+{% set sha = "1f15690caf02ed9c45d088536bee22d10a41d1954077e884b3cdcc361d422904" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anuga-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<3.6]
+
+requirements:
```

Comment:
It looks like there are too many dependencies in each requirements sub-section.

The build section is for build tools like compilers, cmake, etc. These packages must match the host platform when cross compiling.

The host section is for compiled libraries and python build requirements like python, setuptools, libpython, sometimes numpy (if compiled against their c API).

The run section is for packages only needed at runtime like scipy, matplotlib. Cython is rarely needed here.

Packages only needed to run the tests, go in a separate test/requires section. i.e. pytest

Reply:
@carterbox  thanks. I have tried to minimize the requirements as suggested. Just pushed changes so hopefully I have left all the requirements.